### PR TITLE
TOP - Eliminar contrarreferida

### PR DESCRIPTION
**Pull request by Fabio-Ramirez**
### Requerimiento
 
https://proyectos.andes.gob.ar/browse/TOP-153

### Funcionalidad desarrollada 

1. En solicitudes de entrada, se elimina el badge de "contrarreferida".

### Requiere actualizaciones en la base de datos

- [x] Si
- [ ] No

### CODE REVIEW

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9542](http://condorpiedra.andes.gob.ar:85/9542/mochawesome.html)
CYPRESS RUN: [8669](https://dashboard.cypress.io/projects/xr7gft/runs/8669/specs)
TEST START: 2023-09-04T18:01:40.312Z
TOTAL: 403
SUCCESS: 398
FAIL: 3
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9543](http://condorpiedra.andes.gob.ar:85/9543/mochawesome.html)
CYPRESS RUN: [8670](https://dashboard.cypress.io/projects/xr7gft/runs/8670/specs)
TEST START: 2023-09-04T18:19:27.680Z
TOTAL: 357
SUCCESS: 346
FAIL: 9
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9545](http://condorpiedra.andes.gob.ar:85/9545/mochawesome.html)
CYPRESS RUN: [8672](https://dashboard.cypress.io/projects/xr7gft/runs/8672/specs)
TEST START: 2023-09-04T18:40:16.332Z
TOTAL: 403
SUCCESS: 399
FAIL: 2
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9546](http://condorpiedra.andes.gob.ar:85/9546/mochawesome.html)
CYPRESS RUN: [8673](https://dashboard.cypress.io/projects/xr7gft/runs/8673/specs)
TEST START: 2023-09-04T19:04:10.135Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

File: cypress/e2e/apps/solicitud/responderSolicitud.spec.js [hunk 1]
```diff
@@ -11,15 +11,15 @@ context('RESPONDER SOLICITUD', () => {
...
+    beforeEach(() => {
         cy.goto('/solicitudes', token);
         cy.server();
         cy.route('GET', '**/api/core/tm/profesionales**').as('getProfesional');
         cy.route('GET', '**/api/modules/rup/prestaciones/solicitudes**').as('solicitudes');
         cy.route('PATCH', '**/api/modules/rup/prestaciones/**').as('patchSolicitud');
     });
 
-    it('responder', () => {
+    it.only('responder', () => {
```
Comment (MarianoCampetella) [inline cypress/e2e/apps/solicitud/responderSolicitud.spec.js:22]: Remover el .only

**Review by MarianoCampetella — CHANGES_REQUESTED** [review]

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9568](http://condorpiedra.andes.gob.ar:85/9568/mochawesome.html)
CYPRESS RUN: [8691](https://dashboard.cypress.io/projects/xr7gft/runs/8691/specs)
TEST START: 2023-09-06T15:52:03.347Z
TOTAL: 367
SUCCESS: 358
FAIL: 7
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9575](http://condorpiedra.andes.gob.ar:85/9575/mochawesome.html)
CYPRESS RUN: [8696](https://dashboard.cypress.io/projects/xr7gft/runs/8696/specs)
TEST START: 2023-09-07T11:53:35.163Z
TOTAL: 403
SUCCESS: 400
FAIL: 1
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9577](http://condorpiedra.andes.gob.ar:85/9577/mochawesome.html)
CYPRESS RUN: [8698](https://dashboard.cypress.io/projects/xr7gft/runs/8698/specs)
TEST START: 2023-09-07T12:42:19.062Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

File: cypress/e2e/apps/solicitud/responderSolicitud.spec.js (hunk 1 above)
Comment (Fabio-Ramirez) [reply cypress/e2e/apps/solicitud/responderSolicitud.spec.js]: Corregido compa!

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9605](http://condorpiedra.andes.gob.ar:85/9605/mochawesome.html)
CYPRESS RUN: [8721](https://dashboard.cypress.io/projects/xr7gft/runs/8721/specs)
TEST START: 2023-09-12T19:04:49.259Z
TOTAL: 353
SUCCESS: 349
FAIL: 2
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9607](http://condorpiedra.andes.gob.ar:85/9607/mochawesome.html)
CYPRESS RUN: [8723](https://dashboard.cypress.io/projects/xr7gft/runs/8723/specs)
TEST START: 2023-09-13T11:45:44.366Z
TOTAL: 95
SUCCESS: 0
FAIL: 95
SKIPPED: 0
:warning::warning::warning::warning:
Hay archivos que no corrieron. Vuelva a intentar.

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9608](http://condorpiedra.andes.gob.ar:85/9608/mochawesome.html)
CYPRESS RUN: [8724](https://dashboard.cypress.io/projects/xr7gft/runs/8724/specs)
TEST START: 2023-09-13T12:01:51.941Z
TOTAL: 403
SUCCESS: 397
FAIL: 4
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9640](http://condorpiedra.andes.gob.ar:85/9640/mochawesome.html)
CYPRESS RUN: [8754](https://dashboard.cypress.io/projects/xr7gft/runs/8754/specs)
TEST START: 2023-09-19T13:46:13.784Z
TOTAL: 402
SUCCESS: 398
FAIL: 2
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9641](http://condorpiedra.andes.gob.ar:85/9641/mochawesome.html)
CYPRESS RUN: [8755](https://dashboard.cypress.io/projects/xr7gft/runs/8755/specs)
TEST START: 2023-09-19T14:37:40.301Z
TOTAL: 403
SUCCESS: 400
FAIL: 1
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9642](http://condorpiedra.andes.gob.ar:85/9642/mochawesome.html)
CYPRESS RUN: [8756](https://dashboard.cypress.io/projects/xr7gft/runs/8756/specs)
TEST START: 2023-09-19T14:52:08.259Z
TOTAL: 403
SUCCESS: 400
FAIL: 1
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9643](http://condorpiedra.andes.gob.ar:85/9643/mochawesome.html)
CYPRESS RUN: [8757](https://dashboard.cypress.io/projects/xr7gft/runs/8757/specs)
TEST START: 2023-09-19T15:33:17.309Z
TOTAL: 403
SUCCESS: 400
FAIL: 1
SKIPPED: 2

File: cypress/e2e/apps/solicitud/responderSolicitud.spec.js [hunk 2]
```diff
@@ -30,11 +29,10 @@ context('RESPONDER SOLICITUD', () => {
             expect(xhr.status).to.be.eq(200);
             expect(xhr.response.body.estados[1].observaciones).to.be.eq('No se acepta por ...');
             expect(xhr.response.body.solicitud.historial[i].observaciones).to.be.eq('No se acepta por ...');
-        
+
         });
         cy.wait('@solicitudes').then((xhr) => {
             expect(xhr.status).to.be.eq(200);
         });
-        cy.get('plex-badge').contains('CONTRARREFERIDA');
```
Comment (MarianoCampetella) [inline cypress/e2e/apps/solicitud/responderSolicitud.spec.js:38]: Eliminar el badge de 'CONTRAREFERIDA' no seria la solución adecuada ya que se debería chequear cual es el estado en el que tiene que quedar. Dejar el badge con su estado correspondiente

**Review by MarianoCampetella — CHANGES_REQUESTED** [review]

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9670](http://condorpiedra.andes.gob.ar:85/9670/mochawesome.html)
CYPRESS RUN: [8784](https://dashboard.cypress.io/projects/xr7gft/runs/8784/specs)
TEST START: 2023-09-27T15:13:23.306Z
TOTAL: 403
SUCCESS: 400
FAIL: 1
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9672](http://condorpiedra.andes.gob.ar:85/9672/mochawesome.html)
CYPRESS RUN: [8786](https://dashboard.cypress.io/projects/xr7gft/runs/8786/specs)
TEST START: 2023-09-27T15:42:54.259Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

File: cypress/e2e/apps/solicitud/responderSolicitud.spec.js (hunk 2 above)
Comment (Fabio-Ramirez) [reply cypress/e2e/apps/solicitud/responderSolicitud.spec.js:38]: Tal como lo charlamos campe, esta observación seria parte de otra tarea donde se crearía un caso de test.

**Review by MarianoCampetella — APPROVED** [review]

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9838](http://condorpiedra.andes.gob.ar:85/9838/mochawesome.html)
CYPRESS RUN: [8946](https://dashboard.cypress.io/projects/xr7gft/runs/8946/specs)
TEST START: 2023-10-30T13:09:52.815Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

File: cypress/e2e/apps/solicitud/responderSolicitud.spec.js (hunk 2 above)
Comment (negro89) [reply cypress/e2e/apps/solicitud/responderSolicitud.spec.js:38]: consulta chicos, esta tarea ya está creada? Sino deberíamos crearla.

Comment (MarianoCampetella) [reply cypress/e2e/apps/solicitud/responderSolicitud.spec.js:38]: Acá se encuentra dicha tarea: https://proyectos.andes.gob.ar/browse/TOP-171

Comment (Fabio-Ramirez) [reply cypress/e2e/apps/solicitud/responderSolicitud.spec.js:38]: Hola chicos, ya estaba creada y asignada en el planing: https://proyectos.andes.gob.ar/browse/TOP-167

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9860](http://condorpiedra.andes.gob.ar:85/9860/mochawesome.html)
CYPRESS RUN: [8967](https://dashboard.cypress.io/projects/xr7gft/runs/8967/specs)
TEST START: 2023-11-01T16:44:30.963Z
TOTAL: 403
SUCCESS: 399
FAIL: 2
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9862](http://condorpiedra.andes.gob.ar:85/9862/mochawesome.html)
CYPRESS RUN: [8969](https://dashboard.cypress.io/projects/xr7gft/runs/8969/specs)
TEST START: 2023-11-01T18:06:43.684Z
TOTAL: 403
SUCCESS: 399
FAIL: 2
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9896](http://condorpiedra.andes.gob.ar:85/9896/mochawesome.html)
CYPRESS RUN: [9003](https://dashboard.cypress.io/projects/xr7gft/runs/9003/specs)
TEST START: 2023-11-10T14:49:22.382Z
TOTAL: 403
SUCCESS: 400
FAIL: 1
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9898](http://condorpiedra.andes.gob.ar:85/9898/mochawesome.html)
CYPRESS RUN: [9005](https://dashboard.cypress.io/projects/xr7gft/runs/9005/specs)
TEST START: 2023-11-10T15:08:49.054Z
TOTAL: 360
SUCCESS: 351
FAIL: 7
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9900](http://condorpiedra.andes.gob.ar:85/9900/mochawesome.html)
CYPRESS RUN: [9007](https://dashboard.cypress.io/projects/xr7gft/runs/9007/specs)
TEST START: 2023-11-10T16:05:32.078Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9945](http://condorpiedra.andes.gob.ar:85/9945/mochawesome.html)
CYPRESS RUN: [9052](https://dashboard.cypress.io/projects/xr7gft/runs/9052/specs)
TEST START: 2023-11-22T16:23:53.693Z
TOTAL: 403
SUCCESS: 392
FAIL: 9
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9951](http://condorpiedra.andes.gob.ar:85/9951/mochawesome.html)
CYPRESS RUN: [9058](https://dashboard.cypress.io/projects/xr7gft/runs/9058/specs)
TEST START: 2023-11-22T19:12:28.252Z
TOTAL: 403
SUCCESS: 391
FAIL: 10
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9953](http://condorpiedra.andes.gob.ar:85/9953/mochawesome.html)
CYPRESS RUN: [9060](https://dashboard.cypress.io/projects/xr7gft/runs/9060/specs)
TEST START: 2023-11-23T12:05:42.351Z
TOTAL: 403
SUCCESS: 392
FAIL: 9
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9954](http://condorpiedra.andes.gob.ar:85/9954/mochawesome.html)
CYPRESS RUN: [9061](https://dashboard.cypress.io/projects/xr7gft/runs/9061/specs)
TEST START: 2023-11-23T12:22:18.725Z
TOTAL: 403
SUCCESS: 392
FAIL: 9
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9955](http://condorpiedra.andes.gob.ar:85/9955/mochawesome.html)
CYPRESS RUN: [9062](https://dashboard.cypress.io/projects/xr7gft/runs/9062/specs)
TEST START: 2023-11-23T13:34:29.226Z
TOTAL: 403
SUCCESS: 392
FAIL: 9
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9968](http://condorpiedra.andes.gob.ar:85/9968/mochawesome.html)
CYPRESS RUN: [9079](https://dashboard.cypress.io/projects/xr7gft/runs/9079/specs)
TEST START: 2023-11-27T12:08:27.253Z
TOTAL: 403
SUCCESS: 400
FAIL: 1
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9976](http://condorpiedra.andes.gob.ar:85/9976/mochawesome.html)
CYPRESS RUN: [9087](https://dashboard.cypress.io/projects/xr7gft/runs/9087/specs)
TEST START: 2023-11-27T14:40:52.407Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [10051](http://condorpiedra.andes.gob.ar:85/10051/mochawesome.html)
CYPRESS RUN: [9160](https://dashboard.cypress.io/projects/xr7gft/runs/9160/specs)
TEST START: 2023-12-13T14:58:55.224Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [10248](http://condorpiedra.andes.gob.ar:85/10248/mochawesome.html)
CYPRESS RUN: [9355](https://dashboard.cypress.io/projects/xr7gft/runs/9355/specs)
TEST START: 2024-01-12T13:02:55.556Z
TOTAL: 91
SUCCESS: 0
FAIL: 91
SKIPPED: 0
:warning::warning::warning::warning:
Hay archivos que no corrieron. Vuelva a intentar.

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [10256](http://condorpiedra.andes.gob.ar:85/10256/mochawesome.html)
CYPRESS RUN: [9363](https://dashboard.cypress.io/projects/xr7gft/runs/9363/specs)
TEST START: 2024-01-12T18:58:30.528Z
TOTAL: 365
SUCCESS: 348
FAIL: 14
SKIPPED: 3

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [10271](http://condorpiedra.andes.gob.ar:85/10271/mochawesome.html)
CYPRESS RUN: [9378](https://dashboard.cypress.io/projects/xr7gft/runs/9378/specs)
TEST START: 2024-01-15T15:36:51.132Z
TOTAL: 401
SUCCESS: 398
FAIL: 0
SKIPPED: 3

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [10342](http://condorpiedra.andes.gob.ar:85/10342/mochawesome.html)
CYPRESS RUN: [9449](https://dashboard.cypress.io/projects/xr7gft/runs/9449/specs)
TEST START: 2024-01-25T18:39:30.940Z
TOTAL: 401
SUCCESS: 398
FAIL: 0
SKIPPED: 3

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [10427](http://condorpiedra.andes.gob.ar:85/10427/mochawesome.html)
CYPRESS RUN: [9532](https://dashboard.cypress.io/projects/xr7gft/runs/9532/specs)
TEST START: 2024-02-02T11:41:51.028Z
TOTAL: 402
SUCCESS: 398
FAIL: 1
SKIPPED: 3

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [10429](http://condorpiedra.andes.gob.ar:85/10429/mochawesome.html)
CYPRESS RUN: [9534](https://dashboard.cypress.io/projects/xr7gft/runs/9534/specs)
TEST START: 2024-02-02T12:42:29.154Z
TOTAL: 334
SUCCESS: 329
FAIL: 2
SKIPPED: 3

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [10439](http://condorpiedra.andes.gob.ar:85/10439/mochawesome.html)
CYPRESS RUN: [9544](https://dashboard.cypress.io/projects/xr7gft/runs/9544/specs)
TEST START: 2024-02-05T13:49:03.240Z
TOTAL: 399
SUCCESS: 397
FAIL: 0
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [10459](http://condorpiedra.andes.gob.ar:85/10459/mochawesome.html)
CYPRESS RUN: [9564](https://dashboard.cypress.io/projects/xr7gft/runs/9564/specs)
TEST START: 2024-02-07T13:04:32.683Z
TOTAL: 399
SUCCESS: 397
FAIL: 0
SKIPPED: 2

**Pull request merged**